tests/conftest.py

```diff
@@ -87,3 +87,813 @@ def teardown():
     app.test_request_context().push()
 
     return app
+
+
+@pytest.fixture(scope="function")
+def pure_record_merged():
+    """A Pure record with all possible attributes."""
+    return {
```

Comment:
would it be possible to create such big json objects in a separated '*.json' file and load that file?

Reply:
See https://github.com/tu-graz-library/invenio-rdm-pure/pull/33/commits/4909820b9e0e25d77f1c697b3a90252564a593b4.

Reply:
yes, i saw it afterwards. could you please then through out those unnecessary commits

Reply:
As soon as https://github.com/tu-graz-library/invenio-rdm-pure/pull/32 is merged, this branch will be rebased and the commits will be gone.

Reply:
I have removed the commits of https://github.com/tu-graz-library/invenio-rdm-pure/pull/32.